src/PAModel/packages.lock.json

```diff
@@ -30,16 +30,15 @@
       },
       "System.Text.Json": {
         "type": "Direct",
-        "requested": "[9.0.0, )",
-        "resolved": "9.0.0",
-        "contentHash": "js7+qAu/9mQvnhA4EfGMZNEzXtJCDxgkgj8ohuxq/Qxv+R56G+ljefhiJHOxTNiw54q8vmABCWUwkMulNdlZ4A==",
+        "requested": "[8.0.0, )",
+        "resolved": "8.0.0",
```

Comment:
Hmm.  Despite the csproj explicitly asking for 8.0.5, looks like the lockfile is still grabbing 8.0.0?